compiler/packagehandling.nim

```diff
@@ -18,7 +18,9 @@ iterator myParentDirs(p: string): string =
 proc resetPackageCache*(conf: ConfigRef) =
   conf.packageCache = newPackageCache()
 
-proc getPackageName*(conf: ConfigRef; path: string): string =
+when false:
+ proc getPackageName*(conf: ConfigRef; path: string): string =
```

Comment:
Indenting here looks wrong, it should be 2 spaces where this appears to just be 1...

Reply:
In fact, why not just remove this code instead of disabling it?

Reply:
fixed